Core/Core/Models/Assignment.swift

```diff
@@ -71,7 +73,7 @@ extension Assignment {
         details = item.description
         pointsPossible = item.points_possible
         dueAt = item.due_at
-        dueAtOrder = item.due_at == nil ? "z" : "a"
+        dueAtSortNilsAtBottom = item.due_at == nil ? Date.distantFuture : item.due_at!
```

Comment:
`item.due_at ?? .distantFuture` to avoid a force unwrap (I know it's perfectly safe here though)

Reply:
ha, not sure what i was thinking, good catch